ui/framework/src/ui-framework/imodel-components/VisibilityTreeEventHandler.ts

```diff
@@ -204,10 +210,23 @@ export class VisibilityTreeEventHandler extends UnifiedSelectionTreeEventHandler
 
   private createCheckboxInfo(status: VisibilityStatus): CheckBoxInfo {
     return {
-      state: status.isDisplayed ? CheckBoxState.On : CheckBoxState.Off,
+      state: visibilityStateToCheckboxState(status.state),
       isDisabled: status.isDisabled || false,
       isVisible: true,
       tooltip: status.tooltip,
     };
   }
 }
+
+// istanbul ignore next

```

Comment:
do we really need to ignore this?

Reply:
Move it to `case 'partial':` as it is not supported in Models or Category tree and I don't think it's worth to write tests just for this function.